src/finch/interface/lazy.py

```diff
@@ -393,3 +393,79 @@ def positive(x) -> LazyTensor:
 
 def negative(x) -> LazyTensor:
     return elementwise(operator.neg, defer(x))
+
+
+def any(
+    x,
+    /,
+    *,
+    axis: int | tuple[int, ...] | None = None,
+    keepdims: bool = False,
+    init=None,
+):
+    """
+    Test whether any element of input array ``x`` along given axis is True.
+    """
+    if init is None:
+        init = init_value(builtins.any, x)
```

Comment:
I think you mean to use `operator.or_`. Does the array api spec allow an `init` for this function?

Reply:
This is not resolved, the reduce function already defines an init. just pass init unmodified.